lua/gruvbox-baby/theme.lua

```diff
@@ -133,9 +133,10 @@ function M.setup(config)
     ["@variable"] = { fg = c.light_blue, style = config.variable_style },
     ["@variable.builtin"] = { fg = c.blue_gray },
     ["@text"] = { fg = c.soft_yellow },
-    ["@text.strong"] = { fg = c.soft_yellow },
-    ["@text.emphasis"] = { fg = c.soft_yellow },
-    ["@text.underline"] = { fg = c.soft_yellow },
+    ["@text.strong"] = { fg = c.soft_yellow, style = "bold" },
+    ["@text.emphasis"] = { fg = c.soft_yellow, style = "italic" },
+    ["@text.underline"] = { fg = c.soft_yellow, style = "underline" },
+		["@text.strike"] = { fg = c.medium_gray, style = "strikethrough" },
```

Comment:
`medium_gray` isn't consistent with the other text nodes. But I felt it was a sensible choice.